extensions/binding/org.eclipse.smarthome.binding.wemo/ESH-INF/thing/thing-types.xml

```diff
@@ -144,6 +145,13 @@
         <state readOnly="true"></state>
     </channel-type>
     
+     <channel-type id="lastMotionDetected" advanced="true">
+        <item-type>DateTime</item-type>
+        <label>Last Activity (Date/Time)</label>
+        <description>Date/time when last motion was detected</description>
+        <category>Date</category>
```

Comment:
Hm, as there is no item of type DateTime listed at all, what category would you suggest.